src/Parser/DeathKnight/Frost/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,125 @@
+
+class Checklist extends CoreChecklist {
+  static dependencies = {
+    abilities: Abilities,
+    castEfficiency: CastEfficiency,
+    combatants: Combatants,
+    legendaryCountChecker: LegendaryCountChecker,
+    legendaryUpgradeChecker: LegendaryUpgradeChecker,
+    prePotion: PrePotion,
+    alwaysBeCasting: AlwaysBeCasting,
+    enchantChecker: EnchantChecker,
+    runicPowerDetails: RunicPowerDetails,
+    runeTracker: RuneTracker,
+  };
+
+  rules = [
+    new Rule({
+      name: 'Use core spells as often as possible',
+      description: <Wrapper>Spells with short, static cooldowns like <SpellLink id={SPELLS.REMORSELESS_WINTER.id}/> and <SpellLink id={SPELLS.CHAINS_OF_ICE.id}/>(when using Cold Heart) should be used as often as possible</Wrapper>,
+      requirements: () => {
+        const combatant = this.combatants.selected;
+        return [
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.REMORSELESS_WINTER,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.CHAINS_OF_ICE,
+            when: combatant.hasChest(ITEMS.COLD_HEART.id),
+          }),
+          new GenericCastEfficiencyRequirement({
+              spell: SPELLS.HORN_OF_WINTER_TALENT,
+              when: combatant.hasTalent(SPELLS.HORN_OF_WINTER_TALENT.id),
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: 'Use cooldowns as often as possible',
+      description: 'You should aim to use your cooldowns as often as you can to maximize your damage output',
+      requirements: () => {
+        const combatant = this.combatants.selected;
+        return [
+            new GenericCastEfficiencyRequirement({
+              spell: SPELLS.PILLAR_OF_FROST,
+            }),
+            new GenericCastEfficiencyRequirement({
+              spell: SPELLS.OBLITERATION_TALENT,
+              when: combatant.hasTalent(SPELLS.OBLITERATION_TALENT.id),
+            }),
+            new GenericCastEfficiencyRequirement({
+              spell: SPELLS.BREATH_OF_SINDRAGOSA_TALENT,
+              when: combatant.hasTalent(SPELLS.BREATH_OF_SINDRAGOSA_TALENT.id),
+            }),
+            new GenericCastEfficiencyRequirement({
+              spell: SPELLS.EMPOWER_RUNE_WEAPON,
+            }),
+            new GenericCastEfficiencyRequirement({
+                spell: SPELLS.SINDRAGOSAS_FURY_ARTIFACT,
+            }),
+          ];
+      },
+    }),
+    new Rule({
+      name: 'Try to avoid being inactive for a large portion of the fight',
+      description: <Wrapper>While some downtime is inevitable in fights with movement, you should aim to reduce downtime to prevent capping Runes.  You can reduce downtime by casting ranged abilities like <SpellLink id={SPELLS.FROST_STRIKE_CAST.id}/></Wrapper>,
+      requirements: () => {
+        return [
+          new Requirement({
+            name: 'Downtime',
+            check: () => this.alwaysBeCasting.downtimeSuggestionThresholds,
```

Comment:
the downtime suggestion will likely be a bit off since ability.js seems pretty short atm.

Reply:
I thought downtime came from AlwaysBeCasting.js, not Abilities.js.

Reply:
You're right, I didn't check but frost does indeed have the GCDs set in ABC so it should be correct. That way of doing it is deprecated though, ideally it should be set in Abilities with `isOnGCD: true,`